### ISSUE TITLE
saved transfer models disappear after refreshing page

### ISSUE THREAD

**Issue by pejoujarrar** [tensorflow/tfjs#1659]
To get help from the community, we encourage using Stack Overflow and the [`tensorflow.js`](https://stackoverflow.com/questions/tagged/tensorflow.js) tag.

#### TensorFlow.js version
1.1.2
#### Browser version
Firefox Quantum 67.0.1
#### Describe the problem or feature request
After reating a model and collecting data, then training the model and saving it:

`  async trainModel(){
        await transferRecognizer.train({
            epochs: 100,
            callback: {
                onEpochEnd: async (epoch, logs) => {
                    console.log(`Epoch ${epoch}: loss=${logs.loss}, accuracy=${logs.acc}`);
                }
            }
        }).then(async ()=>{
            await transferRecognizer.save();
        });
    }`
After refreshing the page and trying to load the model this way: 
`
 console.log(speechCommands.listSavedTransferModels());
        await transferRecognizer.load().then(async ()=>{
            await transferRecognizer.listen(result => {
                // - result.scores contains the scores for the new vocabulary, which
                //   can be checked with:
                console.log(result.scores);
                const words = transferRecognizer.wordLabels();
                //console.log(words);
                // `result.scores` contains the scores for the new words, not the original
                // words.
                var maxScoreIndex = 0;
                for(var i=1; i<result.scores.length;i++){
                    if(result.scores[maxScoreIndex] < result.scores[i]){
                        maxScoreIndex = i;
                    }
                }
                this.setState({command : transferRecognizer.wordLabels()[maxScoreIndex]});
                console.log(transferRecognizer.wordLabels()[maxScoreIndex]);

            }, {probabilityThreshold: 0.75});
        });
`
the Model doesn't exist anymore in the listSavedTransferModels.

Is there any way to store the model into a file locally?

**Comment by caisq** [comment]
@pejoujarrar Have you tried it in other browsers?

**Comment by pejoujarrar** [comment]
@caisq I actually see where is the problem, after training network, the trained model (parameters) is stored as an indexeddb, so it's related to the browser and also the client  connected to the server. how can i implement the same model for all clients (instead of training a model for each indexdb session).

How can i store the trained model in the local server permanently?

**Comment by caisq** [comment]
@pejoujarrar As you pointed out, the `save()` method is for a local browser environment only. You can do the following to save the trained model to a centralized location. 

1. Use the `IOHandler` option of the `save()` method to get an in-memory representation of the model artifacts. See https://github.com/tensorflow/tfjs-models/blob/master/speech-commands/src/types.ts#L327. You can use tf.io.withHandler() to do that. See https://github.com/tensorflow/tfjs-core/blob/master/src/io/passthrough.ts#L96. The JSON and binary weight artifacts can be uploaded to your backend storage of choice, e.g., Google Cloud Storage or AWS S3 or custom servers.
2. During load time, use model.json, and metadata.json URLs when you call `speechCommands.create()`. See the `customModelURL` and `customMetadataURL` arguments of the method: https://github.com/tensorflow/tfjs-models/blob/master/speech-commands/src/index.ts#L49
